### DISCUSSION TITLE
Mouse or touchscreen input are not working to move the model on Samsung Internet browser

### DISCUSSION THREAD

**Discussion by naurojr** [google/model-viewer#4020] [Q&A]
### Description

I'm testing Model Viewer using a Samsung Display with Tizen and Samsung Internet browser. 
Although I can see the model and the little hand showing how to move the model, the input is not woking. 
I'm testing with the mouse, the touchscreen, and the keyboard.

#### Live Demo
I'm testing with your page
https://modelviewer.dev/examples/lighthouse.html

### Version
The one on your demo page page

### Browser Affected
- [x] Samsung Internet

### OS

- [x] Tizen

**Comment by elalish** [comment]
Strange; does it repro on the homepage too? https://modelviewer.dev

**Comment by naurojr** [comment]
Samsung Internet browser is limited, so your website structure does not read well. I can only see limited content from the main page. The quick start code and the sample are hidden (not shown) in the browser. But the browser itself supports WebGL. I can see most of the samples on https://webglsamples.org/  or https://aframe.io/

**Comment by elalish** [comment]
Interesting; could you provide some screen recordings perhaps? Do you think this is a Samsung internet TV bug? If you have any thoughts on how to improve it from our side, a PR is always welcome.

**Reply by elalish** [reply]
I'm converting to a discussion since there's nothing actionable on the MV side yet.
